glean/data-review/request.md

```diff
@@ -0,0 +1,108 @@
+
+The objective is to measure Daily Active Users for Sync (DAU).  This is to be an internal metric, emitted from the server of the application. We want to understand the usage of the service per user interaction.  This is important for understanding how many users have Sync enabled and at a more granular level, whether the usage is on Firefox Desktop, Fenix or iOS. Multiple actions across devices or on the same device will be reconciled to count as a single Active User.
+
+2) Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements? Some example responses:
+
+* We need to understand usage patterns of Sync from the server-side.
+* Would provide the most granular measurement of service interaction from the server-side.
+* This is necessary to establish a baseline measurement of Daily, Weekly, and Monthly usage of Sync. Essential KPI metric for Sync.
+* Helps us understand usage trends related to feature changes.
+
+3) What alternative methods did you consider to answer these questions? Why were they not sufficient?
+
+* Previously, this metric was measured by FxA/Mozilla Accounts. Logins were associated with an OAuth Client ID. However, as Relay and other account recovery tools make use of Firefox accounts for their auth, this will alter the reliability of this metric. A user could also make use of the aforementioned services without Sync being enabled, resulting in further inaccuracies.
+* A decision was made to move away from attributing active use through auth tokens towards a metric that counts each time Sync collections are accessed. Links to these decision briefs are cited at the bottom of the data review document.
+
+4) Can current instrumentation answer these questions?
+
+* It does at the current moment, but in a short time will no longer be able to.  As described in #3, we are needing to move the metric measurement to the service itself and from the server.
+
+5) List all proposed measurements and indicate the category of data collection for each measurement, using the [Firefox data collection categories](https://wiki.mozilla.org/Data_Collection) found on the Mozilla wiki.   
+
+**Note that the data steward reviewing your request will characterize your data collection based on the highest (and most sensitive) category.**
+
+<table>
+  <tr>
+    <td>Measurement Name</td>
+    <td>Measurement Description</td>
+    <td>Data Collection Category</td>
+    <td>Tracking Bug #</td>
+  </tr>
+  <tr>
+    <td>sync_event</td>
+    <td>Event to record an instance of sync backend activity initiated by client.</td>
+    <td>Cat 2: Interaction Data</td>
+    <td><a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1923967">https://bugzilla.mozilla.org/show_bug.cgi?id=1923967</a></td>
+  </tr>
+  <tr>
+    <td>hashed_fxa_uid</td>
+    <td>Sync user identifier. Uses `hashed_fxa_uid` for accurate count of sync actions.</td>
```

Comment:
May want to explain what the 'hashed_fxa_uid' is. (e.g. "This is the Firefox Accounts (FxA) User Identifier (UID) value passed through a SHA-256 hash to render a value that is unique, but ensures the privacy of the original UID.") It's usually best to over explain than under explain in these sorts of documents.

Reply:
Absolutely, I'll add this context. Ty